pkg/extender/extender.go

```diff
@@ -847,3 +928,172 @@ func (e *Extender) processCSIExtPodPrioritizeRequest(
 		storklog.PodLog(pod).Errorf("Failed to encode response: %v", err)
 	}
 }
+
+func (e *Extender) processVirtLauncherPodPrioritizeRequest(
+	encoder *json.Encoder,
+	args schedulerapi.ExtenderArgs) error {
+	pod := args.Pod
+
+	vmiName, vmiUID := e.getVMIInfo(pod)
+	if vmiName == "" {
+		return fmt.Errorf("unable to find vmiName for pod: %v", pod.Name)
+	}
+	podBeingLiveMigrated, err := e.getLiveMigrationInfo(pod, vmiUID)
+	if err != nil {
+		return fmt.Errorf("unable to find liveMigrationInfo: %w", err)
+	}
+
+	if podBeingLiveMigrated != nil {
+		storklog.PodLog(pod).Infof("Live VM migration in progress: (vmiName: %v, podBeingLiveMigrated: %v)", vmiName, podBeingLiveMigrated.Name)
+	} else {
+		storklog.PodLog(pod).Infof("No ongoing live VM migration in progress. vmiName: %v", vmiName)
+	}
+
+	vmi, err := kvd.Instance().GetVirtualMachineInstance(context.TODO(), pod.Namespace, vmiName)
+	if err != nil {
+		return fmt.Errorf("unable to find VMI Info: %w", err)
+	}
+	storklog.PodLog(pod).Debugf("VMI is using PVC: %v", vmi.RootDiskPVC)
+
+	pvName, err := e.GetPVNameFromPVC(vmi.RootDiskPVC, pod.Namespace)
+	if err != nil {
+		return fmt.Errorf("unable to inspect PVC: %v err: %w", vmi.RootDiskPVC, err)
```

Comment:
include namespace in the error message

nit: This is subjective so feel free to ignore but I feel that there is no need for extra colons. Better to follow the standard convention for error messages `<msg>: %w", err` e.g `"error inspecting PVC %v: %w", <pvc>, err`